client/src/components/pages/profileBusiness/index.js

```diff
@@ -0,0 +1,274 @@
+import React from 'react'
+import { Form, Input, AutoComplete, Cascader, Select } from 'antd'
+
+import Button from 'components/utils/Button'
+import { residences, BusinesTypeValues } from './static'
+import Options from "components/utils/option"
+import json from './country.json'
+
+const AutoCompleteOption = AutoComplete.Option;
+
+class BusinessForm extends React.Component {
+
+  state = {
+    autoCompleteResultWebsite: [],
+    autoCompleteResultCuntry: [],
+    autoCompleteResultCity: [],
+    country: ''
+  };
+
+  handleWebsiteChange = value => {
+    let autoCompleteResultWebsite;
+    if (!value) {
+      autoCompleteResultWebsite = [];
+    } else {
+      autoCompleteResultWebsite = ['.com', '.org', '.net'].map(domain => `${value}${domain}`);
+    }
+    this.setState({ autoCompleteResultWebsite });
+  };
+
+  handleCuntryChange = value => {
+    let autoCompleteResultCuntry;
+    if (!value) {
+      autoCompleteResultCuntry = [];
+    } else {
+      const countries = Object.keys(json)
+      let data = countries.filter((country) => {
+        return country.toLowerCase().startsWith(value.toLowerCase())
+      })
+      autoCompleteResultCuntry = data.map(domain => `${domain}`);
+    }
+    this.setState(() => {
+      return {
+        autoCompleteResultCuntry,
+        country: value
+      }
+    });
+  };
+
+  handleCityChange = value => {
+    const { country } = this.state
+    let autoCompleteResultCity;
+    if (!value) {
+      autoCompleteResultCity = [];
+    } else {
+      let data = json[country].filter((city) => {
+        return city.toLowerCase().includes(value.toLowerCase());
+      })
+      autoCompleteResultCity = data.map(domain => `${domain}`);
+    }
+    this.setState(() => {
+      return { autoCompleteResultCity }
+    });
```

Comment:
@AbdallahAmmar96 The handleCountryChange comments I added above apply here as well